BUILD.bazel

```diff
@@ -94,6 +95,7 @@ genrule(
     srcs = ["CHANGES"],
     outs = ["build-version.inc"],
     cmd = "SOURCE_DATE_EPOCH=0 $(location update_build_version) $$(dirname $(location CHANGES)) $(location build-version.inc)",
+	cmd_bat = "set SOURCE_DATE_EPOCH=0  && $(location //:update_build_version) \"$(location CHANGES)\\..\" $(location build-version.inc)",
```

Comment:
The formatting seems off.  Please replace the tab with an appropriate number of spaces.